client/post-editor/controller.js

```diff
@@ -203,26 +196,9 @@ async function redirectIfBlockEditor( context, next ) {
 
 	const postType = determinePostType( context );
 	const postId = getPostID( context );
-
-	if ( false === isJetpackMinimumVersion( state, siteId, '7.3-alpha' ) ) {
-		const siteAdminUrl = getSiteAdminUrl( state, siteId );
-		let url = `${ siteAdminUrl }post-new.php?post_type=${ postType }`;
-
-		if ( postId ) {
-			url = `${ siteAdminUrl }post.php?post=${ postId }&action=edit`;
-		}
-
-		if ( isSiteAtomic( state, siteId ) ) {
```

Comment:
Can some clarify why we were using Calypsoify only on Atomic sites? Do we want to avoid Calypsoify on JP sites for any reason?

Reply:
/cc @kwight

Reply:
I think it was because we were given literally one day to implement it for Atomic, so we did the minimum required. There's no reason to _not_ have Calypsoify for JP sites I can think of.

Reply:
So Calypsoify changes include both navigation tweaks and CSS styling, to sort of make it look like Calypso.

The latter is generally not needed for Jetpack sites, since they folks know what wp-admin is.

Reply:
If folks don't have any strong concern against having Calypsoify flows on Jetpack, then I think we should have them so we don't have one more exception to maintain. That's how is currently working after merging this PR, so no more actions are needed.